nixos/modules/services/misc/conduit.nix

```diff
@@ -0,0 +1,118 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.conduit;
+
+  format = pkgs.formats.toml {};
+  namespacedSettings.global = cfg.settings;
+  configFile = format.generate "conduit.toml" namespacedSettings;
+in
+  {
+    options.services.conduit = {
+      enable = mkEnableOption "conduit";
+      iHaveReadTheFederationWarning = mkOption {
```

Comment:
While I appreciate the intent, Conduit plans to release 1.0 this year. What is your upgrade plan for this option once it's become obsolete?

Reply:
The plan is to deprecate that option with a note as usual.